composeml/label_times.py

```diff
@@ -226,3 +226,38 @@ def bin(self, bins, quantiles=False, labels=None, right=True):
 
         label_times.transforms.append(transform)
         return label_times
+
+    def sample(self, n=None, frac=None, random_state=None):
+        if isinstance(n, int):
+            sample = super().sample(n=n, random_state=random_state)
+            return sample
+
+        if isinstance(n, dict):
+            sample_per_label = []
+            for label, n, in n.items():
+                is_label = self[self.name].eq(label)
```

Comment:
can these two lines can just be 

`label  = self[self[self.name] == label]`

i don't think you need to do `.loc` or `index`

Reply:
refactored the lines